pkg/api/handlers/compat/containers_stats.go

```diff
@@ -44,17 +44,11 @@ func StatsContainer(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// If the container isn't running, then let's not bother and return
-	// immediately.
-	state, err := ctnr.State()
+	_, err = ctnr.State()
```

Comment:
You can eliminate this entirely, it was only used for checking define.ContainerStateRunning below.